### PR TITLE
hotfix/CP-6124-app-banners-flutter-error-inflating-class-com-google-a…

### PR DESCRIPTION
**Pull request by unnaticleverpush**
…ndroid-material-tabs-tab-layout

Add try catch finally block for material theme

### CODE REVIEW

File: cleverpush/src/main/java/com/cleverpush/banner/AppBannerPopup.java [hunk 1]
```diff
@@ -218,12 +220,27 @@ public void setOpenedListener(AppBannerOpenedListener openedListener) {
...
-      View layout = activity.getLayoutInflater().inflate(layoutId, null);
-      layout.setOnClickListener(view -> dismiss());
-      return layout;
+      layout = activity.getLayoutInflater().inflate(layoutId, null);
     } catch (Exception exception) {
       Logger.e(TAG, exception.getLocalizedMessage());
+      isExceptionGenerate = true;
+    } finally {
+      try {
+        if (isExceptionGenerate) {
```
Comment (cpKevinrodriguesnobre) [inline cleverpush/src/main/java/com/cleverpush/banner/AppBannerPopup.java:232]: i think we were supposed to check if its the one specific error message, that the customer is facing

Comment (cpKevinrodriguesnobre) [inline cleverpush/src/main/java/com/cleverpush/banner/AppBannerPopup.java:232]: all the stuff in this if condition can be moved to above catch and then you dont need isExceptionGenerate anymore

File: cleverpush/src/main/java/com/cleverpush/inbox/InboxDetailActivity.java [hunk 2]
```diff
@@ -164,12 +166,27 @@ private void init() {
...
-      View layout = activity.getLayoutInflater().inflate(layoutId, null);
-      layout.setOnClickListener(view -> dismiss());
-      return layout;
+      layout = activity.getLayoutInflater().inflate(layoutId, null);
     } catch (Exception exception) {
       Logger.e(TAG, exception.getLocalizedMessage());
+      isExceptionGenerate = true;
+    } finally {
+      try {
+        if (isExceptionGenerate) {
```
Comment (cpKevinrodriguesnobre) [inline cleverpush/src/main/java/com/cleverpush/inbox/InboxDetailActivity.java:178]: same stuff for this, maybe we even want to but this whole try catch into a function not sure

**Review by cpKevinrodriguesnobre — CHANGES_REQUESTED** [review]

File: cleverpush/src/main/res/values/styles.xml [hunk 3]
```diff
@@ -25,4 +25,7 @@
     <style name="alertDialogTheme" parent="ThemeOverlay.AppCompat.Dialog.Alert">
     </style>
 
+    <style name="app_banner" parent="Theme.MaterialComponents.Light.NoActionBar">
```
Comment (gbhrdt) [inline cleverpush/src/main/res/values/styles.xml:28]: Let's rename to `cleverpush_app_banner_theme`

**Review by gbhrdt — CHANGES_REQUESTED** [review]

File: cleverpush/src/main/res/values/styles.xml (hunk 3 above)
Comment (unnaticleverpush) [reply cleverpush/src/main/res/values/styles.xml:28]: ok

**Pull request merged**